KNOWN_ISSUES.md

```diff
@@ -45,12 +45,29 @@
   OKD-specific catalog with community versions of mentioned operators is in progress.
 
 ## [Systemd-resolved is not configured properly during install](https://github.com/openshift/okd/issues/690)
-  **Effected Versions:**  4.7.0-0.okd-2021-06-13-090745 - 4.7.0-0.okd-2021-06-13-090745
+  **Affected Versions:**  4.7.0-0.okd-2021-06-13-090745 - 4.7.0-0.okd-2021-06-13-090745
 
   **Description:** With the newest stable release, systemd-resolved is misconfigured due to a missing directory.  This prevents nodes from installing properly.
   
-  **Workaround** from each affected node do the following
-* sudo su -
-* mkdir /etc/systemd/resolved.conf.d
-* reboot
+  **Workaround:** From each affected node do the following:
+
+  ```
+sudo mkdir -p /etc/systemd/resolved.conf.d && sudo systemctl reboot
+  ```
+
+## [systemd crash with CoreOS stable 34.20210711.3.0](https://github.com/coreos/fedora-coreos-tracker/issues/912)
+
+  **Affected Versions:**  4.7.0-0.okd-2021-08-07-063045 - 4.7.0-0.okd-2021-08-07-063045
+
+  **Description:** Systemd version systemd-248.5-1.fc34.x86_64 may core dump with this message:
```

Comment:
Is there a bug for this crash? How often does it happen?

Reply:
This systemd ABRT happens 100% of the time after an unknown duration (minutes to hours) under moderate load (more than one and less than ten user deployments per node, around a dozen nodes). It's been observed 0% of the time on nodes with light load (a single user deployment per node, less than five nodes). With the issue clearly identified upstream and the fix imminent here, I skipped a local OKD issue that points directly to the CoreOS issue, felt redundant, but content to add one for the record.

Reply:
But is there a bugzilla link? I don't think its a great idea to make people match their symptoms

Reply:
Yes, the Bugzilla link is in the second paragraph.

Reply:
Doh! Okay, LGTM